new-version.sh

```diff
@@ -15,68 +15,74 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-# Usage: $ ./new-version.sh <VERSION>
+# Usage: $ ./new-version.sh <NEW_VERSION> <NEXT_VERSION>
 
 set -e
 
-# PYTHON
+usage() {
+  echo "usage: ./$(basename -- ${0}) <NEW_VERSION> <NEXT_VERSION>"
+  exit 1
+}
+
+readonly SEMVER_REGEX="^[0-9]+(\.[0-9]+){2}(-rc\.[0-9]+)?(-SNAPSHOT)?$" # X.Y.Z
+                                                                        # X.Y.Z-rc.*
+                                                                        # X.Y.Z-SNAPSHOT
+
+# Change working directory to project root
+project_root=$(git rev-parse --show-toplevel)
+cd "${project_root}"
 
 # Verify bump2version is installed
 if [[ ! $(type -P bump2version) ]]; then
- echo "bump2version not installed! Please see https://github.com/c4urself/bump2version#installation"
- exit 1
+  echo "bump2version not installed! Please see https://github.com/c4urself/bump2version#installation"
```

Comment:
It would be nice to also put this requirement in the usage comments, so readers don't have to go through the whole script to understand requirements.